app/src/main/java/org/fossasia/susi/ai/MainActivity.java

```diff
@@ -119,7 +120,10 @@ public boolean onOptionsItemSelected(MenuItem item) {
     @Override
     protected void onResume() {
         super.onResume();
-        etMessage.requestFocus();
+        //etMessage.requestFocus();
```

Comment:
@daminisatya please review the PR before merging as this one breaks the keyboard popping up feature instead of solving it.